src/m365/pp/commands/dataverse/dataverse-table-list.ts

```diff
@@ -59,18 +59,11 @@ class PpDataverseTableListCommand extends PowerPlatformCommand {
     }
 
     const dynamicsApiUrl = await powerPlatform.getDynamicsInstanceApiUrl(args.options.environment, args.options.asAdmin);
```

Comment:
Moved this line into the `try` block so when the request fails, the error is properly logged.